api/client_test.go

```diff
@@ -6,10 +6,11 @@ import (
 
 // TestApiClient tests that the API client connects to the server tester and
 // can call and decode routes correctly.
-func TestIntegrationApiClient(t *testing.T) {
+func TestApiClient(t *testing.T) {
 	if testing.Short() {
 		t.SkipNow()
 	}
+	t.Parallel()
```

Comment:
API tests were running too slowly, and I was relying primarily on API tests because I know that we plan to change more of the renter + contractor internals in the near future (mainly the persist stuff). While I was adding t.Parallel() to everything, I also removed the obsoleted 'Integration' word from the testing names, where I ran into it. I was not super thorough.